tutorials/download_function_calling_llama_2.md

```diff
@@ -0,0 +1,59 @@
+## Download [Function Calling Llama 2](https://huggingface.co/Trelis/Llama-2-7b-chat-hf-function-calling-v2) weights
+
+Llama-7B with function calling is licensed according to the Meta Community license.
+
+Function calling Llama extends the hugging face Llama 2 models with function calling capabilities.
+The model responds with a structured json argument with the function name and arguments.
+
+
+In order to use the checkpoint, download the weights and convert the checkpoint to the lit-gpt format.
+
+```bash
+pip install huggingface_hub
+
+python scripts/download.py --repo_id Trelis/Llama-2-7b-chat-hf-function-calling-v2 --access_token your_hf_token --from_safetensors true
```

Comment:
It doesn't seem like this model is gated

```suggestion
python scripts/download.py --repo_id Trelis/Llama-2-7b-chat-hf-function-calling-v2 --from_safetensors true
```

Reply:
Good catch, copy&paste error